src/transforms/projectionpursuit.jl

```diff
@@ -0,0 +1,196 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    ProjectionPursuit(;tol=1e-6, maxiter=100, deg=5, perc=.9, n=100)
+
+The projection pursuit multivariate transform converts any multivariate distribution into
+the standard multivariate Gaussian distribution.
+
+This iterative algorithm repeatedly finds a direction of projection `α` that maximizes a score of
+non-Gaussianity known as the projection index `I(α)`. The samples projected along `α` are then
+transformed with the [`Quantile`](@ref) transform to remove the non-Gaussian structure. The
+other coordinates in the rotated orthonormal basis `Q = [α ...]` are left untouched.
+
```

Comment:
Maybe it would be better to change the notation of the Q to U as in the paper?

Reply:
I am divided about this one. Let's try to follow the notation of the paper strictly and see if the code becomes too strange. In that case we can try to go back to Q as it is usually the case with QR.